test/linalg/dense.jl

```diff
@@ -676,7 +676,7 @@ end
     a = rand(elty)
     @test exp(a) == exp(a)
     @test isposdef(one(elty))
-    @test sqrtm(a) == sqrt(a)
+    @test sqrt(a) == sqrt(a)
```

Comment:
Could probably drop this test

Reply:
might as well drop the `exp(a) == exp(a)` one too while at it.

Reply:
If those tests fail, we know something more fundamentally, existentially wrong.

Reply:
Well, we do also test `@test true` and `@test !false`, etc.